labs-version-probe: drift is reported to a step summary nobody reads

`labs-version-probe.yml` runs nightly and reports drift to a step summary. Two separate defects, one of which makes a failed probe indistinguishable from a clean one.

## 1. A crashed or network-failed probe reports "everything is current"

Lines 59-73:

```bash
node scripts/check-labs-versions.mjs > /tmp/drift.txt 2>&1 || true
{
  if grep -qE '^DRIFT|behind upstream' /tmp/drift.txt; then
    echo 'Drift detected. Dispatch **Upstream Version Watch** ...'
  else
    echo 'No drift. Every pin and declared floor is current.'
  fi
  ...
} >> "$GITHUB_STEP_SUMMARY"
```

`|| true` discards the exit code, and stderr is redirected *into the file the grep reads*. So any non-drift output — a stack trace, a fetch timeout — fails to match `^DRIFT|behind upstream` and falls into the `else` branch, which asserts every pin is current.

Reproduced with the logic isolated and both controls run:

| input | verdict |
|---|---|
| `DRIFT portless 0.10.0 -> 0.15.5` | Drift detected ✅ (control: detection works) |
| `all pins current` | No drift ✅ (control: clean is clean) |
| `Error: Cannot find module 'scripts/check-labs-versions.mjs'` | **No drift. Every pin and declared floor is current.** ❌ |
| `FetchError: request to registry.npmjs.org failed, reason: ETIMEDOUT` | **No drift. Every pin and declared floor is current.** ❌ |

The network case needs no code change to trigger — it is the ordinary failure path for a job that hits npm and GitHub every night.

Fix: capture the exit code and make could-not-observe its own third outcome, never folded into "clean".

```bash
node scripts/check-labs-versions.mjs > /tmp/drift.txt 2>&1; rc=$?
if [ "$rc" -ne 0 ]; then
  echo 'PROBE FAILED (rc='"$rc"') — drift state UNKNOWN, not clean.'
elif grep -qE '^DRIFT|behind upstream' /tmp/drift.txt; then
  ...
```

## 2. The deliverable is a surface nobody opens

The step summary is the only output — no issue, no PR, no notification. `grep -rl labs-version-probe` finds only the two workflows themselves.

The existing comment defends this, and the reasoning is sound as far as it goes:

> Never fail the nightly run on drift: a red cron nobody can action is a red cron people mute. The step summary is the deliverable.

Agreed that a red cron gets muted. But the current design routes a real finding to a page nobody navigates to, so the outcome is the same as not detecting it.

**Measured consequence.** Probe runs succeeded 2026-08-15, 08-16, 08-17. Across that window `src/skills/portless/references/upstream.md` documented portless's proxy default as "port 1355, no sudo needed" while the installed CLI had defaulted to HTTPS on 443 since 0.13 — five minor versions of drift, through three green probes. That stale doc then cost a real session: it read as the normal path, so a port in a `.localhost` URL looked expected rather than broken (fixed in #3525).

Suggested fix that avoids alert fatigue: on drift, open-or-update **one** long-lived issue (idempotent, edited in place, closed when clean). One notification on state change, not one per night.

## Scope note

These are independent. #1 is a correctness bug worth fixing on its own — right now a green probe is not evidence of anything, which makes #2 worse than it looks: the surface nobody reads may also be lying.

--- a/.github/workflows/labs-version-probe.yml
+++ b/.github/workflows/labs-version-probe.yml
@@ -57,11 +57,22 @@ jobs:
       - name: Probe upstream registries
         run: |
           set -uo pipefail
-          node scripts/check-labs-versions.mjs > /tmp/drift.txt 2>&1 || true
+          # Capture the exit code. `|| true` used to discard it, and stderr is
+          # redirected INTO the file the grep reads, so a stack trace or a fetch
+          # timeout matched neither pattern and fell through to the else branch —
+          # printing "Every pin and declared floor is current" for a probe that
+          # never completed. Could-not-observe is its own outcome, never "clean".
+          node scripts/check-labs-versions.mjs > /tmp/drift.txt 2>&1
+          probe_rc=$?
           {
             echo '## Upstream version probe'
             echo ''
-            if grep -qE '^DRIFT|behind upstream' /tmp/drift.txt; then
+            if [ "$probe_rc" -ne 0 ]; then
+              echo "**Probe FAILED (exit $probe_rc). Drift state is UNKNOWN, not clean.**"
+              echo ''
+              echo 'Nothing was verified this run. Do not read this as "pins are current" —'
+              echo 'read the output below, fix the probe, and re-dispatch.'
+            elif grep -qE '^DRIFT|behind upstream' /tmp/drift.txt; then
               echo 'Drift detected. Dispatch **Upstream Version Watch** to open the bump PR.'
             else
               echo 'No drift. Every pin and declared floor is current.'
@@ -71,5 +82,11 @@ jobs:
             cat /tmp/drift.txt
             echo '```'
           } >> "$GITHUB_STEP_SUMMARY"
-          # Never fail the nightly run on drift: a red cron nobody can action is a red
+          # Never fail the nightly run on DRIFT: a red cron nobody can action is a red
           # cron people mute. The step summary is the deliverable.
+          # A FAILED PROBE is different — it means the check did not run, so it goes
+          # red. That is actionable (fix the probe) and cannot be confused with clean.
+          if [ "$probe_rc" -ne 0 ]; then
+            echo "::error::upstream version probe failed (exit $probe_rc) — drift state unknown"
+            exit "$probe_rc"
+          fi

